src/notifications/NotificationsHandler.tsx

```diff
@@ -255,8 +252,7 @@ export const NotificationsHandler = ({ walletReady }: Props) => {
             })
         )
       );
-      initializeGlobalNotificationSettings();
-      initializeNotificationSettingsForAllAddressesAndCleanupSettingsForRemovedWallets(addresses);
+      initializeNotificationSettingsForAllAddresses(addresses);
```

Comment:
`initializeNotificationSettingsForAllAddresses` will now handle the global notification initialization as well since it's all put together in one payload to the backend